template/Cake.Frosting.Template.nuspec

```diff
@@ -5,8 +5,9 @@
         <description>Cake.Frosting templates for the .NET SDK.</description>
         <summary>Cake (C# Make) is a build automation system with a C# DSL to do things like compiling code, copy files/folders, running unit tests, compress files and build NuGet packages.</summary>
         <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström and contributors</authors>
-        <licenseUrl>https://github.com/cake-build/frosting/blob/develop/LICENSE</licenseUrl>
+        <license type="expression">MIT</license>
         <projectUrl>https://github.com/cake-build/frosting</projectUrl>
+        <icon>cake-medium.png</icon>
```

Comment:
Image would need to be added to repo to be accessible from CI.

Reply:
Looks to me like this was added above, right?

Reply:
Ah correct my mistake.